tutorials/hydra.md

```diff
@@ -0,0 +1,339 @@
+Tutorial: Learning Hydra for configuring ML experiments - Simone Scardapane
+=======================================================
+
+9 minut czytania
+
+Ten post jest przeznaczony jako krótkie, samodzielne wprowadzenie do narzędzia Hydra. Omówiono w nim wiele tematów, w tym jak tworzyć obiekty klas, uruchamiać przeszukiwanie parametrów i sprawdzać poprawność konfiguracji w czasie pracy. Wprowadzenie nie obejmuje pełnego zakresu opcji oferowanych przez Hydrę. W tym celu należy zgłębić [oryginalną dokumentację](https://hydra.cc/docs/next/intro).
+
+> 🔗 Kod dla tego tutoriala jest dostępny na repozytorium GitHub: [https://github.com/sscardapane/hydra-tutorial](https://github.com/sscardapane/hydra-tutorial).
+
+Instalacja i przegląd
+-------------------------
+
+Będziemy korzystać z wersji 1.1 biblioteki Hydra, którą można zainstalować poprzez:
+
+    pip install hydra-core==1.1.0.rc1
```

Comment:
czemu 1.1 skoro stable to 1.2 [https://hydra.cc/docs/intro/](https://hydra.cc/docs/intro/)

Reply:
Zmieniono na wersje stabilną